### ISSUE TITLE
Enumerated sublists with explicit start render wrong in LaTeX

### ISSUE THREAD

**Issue by bgeron** [sphinx-doc/sphinx#4577]
### Problem

The following renders wrong in LaTeX:

```rst
1.  First item

    5.  Fifth subitem
    6.  Sixth subitem

2.  Second item
```

**Expected / HTML rendering**: visually similar to source code

**Wrong / current LaTeX PDF rendering**:

```
1.  First item
    (a) Fifth subitem
    (b) Sixth subitem
5.  Second item
```

The problem is that "Second item" was never labelled "5" in the RST source.

The generated LaTeX source for this is as follows:

```latex
\begin{enumerate}
\item {} 
First item
\begin{enumerate}
\setcounter{enumi}{4}
\item {} 
Fifth subitem

\item {} 
Sixth subitem

\end{enumerate}

\item {} 
Second item

\end{enumerate}
```

**Expected LaTeX output**: ... `\setcounter{enumii}{4}` (rather than `enumi`).

### Environment info
- OS: Ubuntu 17.10
- Python version: 3.6.3
- Sphinx version: 1.6.5

**Comment by bgeron** [comment]
Mildly related: #1431

**Comment by tk0miya** [comment]
Fixed by #4639.
Thank you for reporting!

**Comment by bgeron** [comment]
Thanks for fixing!
